packages/gestalt/src/Colors.css

```diff
@@ -1,10 +1,13 @@
 :root {
   /* primary colors */
-  --red: #e60023;
-  --white: #fff;
-  --lightGray: #efefef;
-  --gray: #767676;
-  --darkGray: #111;
+  --colorRed0: #ff5247;
+  --colorRed100: #e60023;
+  --colorGray0: #fff;
+  --colorGray50: #fff;
```

Comment:
@jennyscript this is so weird - should `--colorGray0` and `--colorGray50` have the exact same value? Can we just keep in `--colorGray0` for now?

Reply:
The rationale for colorGray50 is that the dark mode needs an extra gray to address elevation in an accessible way. Happy to go through the nuances of this if you're interested.

Reply:
@ponori understood - it was just odd to see it here without the dark mode overrides

Reply:
@christianvuerings Yes, it's there for the elevated value like PJ mentioned. You can see where I started adding this to Modal and Contents if that helps explain

Reply:
@christianvuerings Yeah, totally understand. Dark mode just understandably adds some moving pieces to our equation. I wonder if there's ample rationale to slightly darken our elevated surfaces in light mode to have a consistent palette model, but I think that's beyond the scope of this effort. That said, with tokens, it should enable us to make that change more seamlessly in the future